MetaMorpheus/TaskLayer/CalibrationTask/CalibrationTask.cs

```diff
@@ -189,7 +184,28 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
 
             return MyTaskResults;
         }
+        private bool CalibrationHasValue(DataPointAquisitionResults acquisitionResultsFirst, DataPointAquisitionResults acquisitionResultsSecond)
```

Comment:
Add a summary comment explaining how important this method is/what its function is